plugins/thunderbolt/fu-plugin-thunderbolt.c

```diff
@@ -37,6 +38,26 @@ struct FuPluginData {
 	GUdevClient   *udev;
 };
 
+static gboolean
+fu_plugin_thunderbolt_safe_kernel (GError **error)
+{
+#ifdef KERNEL_SAFETY_CHECKS
```

Comment:
I think this is a sane thing to do in all cases; we want to err on being safe -- and 4.13 was released nearly two years ago....